### ISSUE TITLE
Can't create new world.

### ISSUE THREAD

**Issue by Blatts12** [jakimfett/Minechem#808]
When I trying create new world. Without MineChem everything works fine.

Minechem-1.7.10-5.0.5.406

http://pastebin.com/GyjgS8yy

**Comment by Cazadorsniper** [comment]
getting the same error only it didn't crash, hears the fml log (whole log is to big for pastebin)  http://pastebin.com/L6carfjU

**Comment by speakerrob** [comment]
Same prob, I've heard [removing Better Builders Wands](https://github.com/portablejim/BetterBuildersWands/issues/4) stops this problem, but it's definitely a Minechem issue, so it is likely to also occur with other mod combinations as well.

http://pastebin.com/QC2QuZYY

**Comment by Wealthyturtle** [comment]
Yeah Cazador, I've been Trying to Add it to My Modpack, but Apparently, Better Builders Wands is Messing Up Minechem Somehow... And Since Minechem is "Dead", I Hope that the Better Builders Wands Devs Could Handle the Issue?

**Comment by Wealthyturtle** [comment]
https://github.com/iopleke/Minechem/issues/841

**Comment by PegasusEpsilon** [comment]
@speakerrob I haven't seen it occur with any other mods. I have no idea what BBW is doing to screw with Minechem, but it seems (from my anecdotal experience) to be a problem between exactly those two, and no others.

I'd be interested in being proven wrong, though.

**Comment by PegasusEpsilon** [comment]
@Wealthyturtle It's a guess as good as any, for all I know. An odd interaction, regardless.
